OpenRA.Mods.Common/Traits/Render/Contrail.cs

```diff
@@ -33,8 +33,11 @@ public class ContrailInfo : ConditionalTraitInfo, Requires<BodyOrientationInfo>
 		[Desc("Time (in ticks) after which the line should appear. Controls the distance to the actor.")]
 		public readonly int TrailDelay = 0;
 
-		[Desc("Thickness of the emitted line.")]
-		public readonly WDist TrailWidth = new WDist(64);
+		[Desc("Thickness of the emitted line at the start of the contrail.")]
+		public readonly WDist StartWidth = new WDist(64);
+
+		[Desc("Thickness of the emitted line at the end of the contrail. Set to StartWidth if undefined")]
```

Comment:
Use `nameof(StartWidth)` in the description.
"Will default to ... if left undefined."